build/build-manifest.php

```diff
@@ -51,8 +51,10 @@
 $manifest = [];
 foreach (glob(__DIR__ . '/../src/data/**/**/api-2.json') as $file) {
     $model = json_decode(file_get_contents($file), true);
+    preg_match('@src/data/([^/]+)/[0-9]{4}-[0-9]{2}-[0-9]{2}/api-2.json$@', $file, $matches);
+    $identifier = $matches[1];
```

Comment:
That's true, I'm also thinking about testing it throughly, will follow up with a new commit.